blueflood-core/src/main/java/com/rackspacecloud/blueflood/io/AstyanaxReader.java

```diff
@@ -47,8 +48,8 @@
 import org.slf4j.LoggerFactory;
 
 import java.io.IOException;
-import java.sql.Connection;
 import java.util.*;
+import java.util.concurrent.*;
 
 public class AstyanaxReader extends AstyanaxIO {
```

Comment:
So it sounds like I can ignore all of the changes in this file, is that correct?